chainerrl/wrappers/vector_frame_stack.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+
+from collections import deque
+
+from gym import spaces
+import numpy as np
+
+from chainerrl.wrappers.atari_wrappers import LazyFrames
+from chainerrl.env import VectorEnv
+
+
+class VectorEnvWrapper(VectorEnv):
+    """VectorEnv analog to gym.Wrapper."""
+
+    def __init__(self, env):
+        self.env = env
+        self.action_space = self.env.action_space
+        self.observation_space = self.env.observation_space
+
+    def __getattr__(self, name):
+        if name.startswith('_'):
+            raise AttributeError(
+                "attempted to get missing private attribute '{}'".format(name))
+        return getattr(self.env, name)
+
+    def step(self, action):
+        return self.env.step(action)
+
+    def reset(self, **kwargs):
+        return self.env.reset(**kwargs)
+
+    def render(self, mode='human', **kwargs):
+        return self.env.render(mode, **kwargs)
+
+    def close(self):
+        return self.env.close()
+
+    def seed(self, seed=None):
+        return self.env.seed(seed)
+
+    def compute_reward(self, achieved_goal, desired_goal, info):
+        return self.env.compute_reward(achieved_goal, desired_goal, info)
+
+    def __str__(self):
+        return '<{}{}>'.format(type(self).__name__, self.env)
+
+    def __repr__(self):
+        return str(self)
+
+    @property
+    def unwrapped(self):
+        return self.env.unwrapped
```

Comment:
https://github.com/chainer/chainerrl/blob/master/chainerrl/envs/multiprocess_vector_env.py#L47
`MultiprocessVectorEnv` inherits `VectorEnv`, which does not have property `@unwrapped`.
It seems `vec_env` created at https://github.com/chainer/chainerrl/pull/443/files#diff-02e7d718fe75bb8ca034c551bd27ee05R166 will raise error when a user tries to access `vec_env.unwrapped`.